runtime/redirector/forwarders.m4

```diff
@@ -269,18 +269,26 @@ _X(JVM_PrintStackTrace,JNICALL,true,jobject ,jint arg0, jint arg1, jint arg2)
 _X(JVM_SetField,JNICALL,true,jobject ,jint arg0, jint arg1, jint arg2, jint arg3)
 _X(JVM_SetPrimitiveField,JNICALL,true,jobject ,jint arg0, jint arg1, jint arg2, jint arg3, jint arg4, jint arg5)
 _X(JVM_SetNativeThreadName,JNICALL,true,void ,jint arg0, jobject arg1, jstring arg2)
-_IF([JAVA_SPEC_VERSION >= 9],
+_IF([JAVA_SPEC_VERSION >= 9 && JAVA_SPEC_VERSION < 15],
```

Comment:
Parenthese would be good here. Also, writing as
```
(9 <= JAVA_SPEC_VERSION) && (JAVA_SPEC_VERSION < 15)
```
reads better (IMO): `JAVA_SPEC_VERSION` is between 9 and 15.

Reply:
Please update similar compound conditions below.